src/pages/workspace/accounting/xero/export/XeroBankAccountSelectPage.tsx

```diff
@@ -28,7 +28,7 @@ function XeroBankAccountSelectPage({policy}: WithPolicyConnectionsProps) {
 
     const {config} = policy?.connections?.xero ?? {};
     const xeroSelectorOptions = useMemo<SelectorType[]>(
-        () => getXeroBankAccountsWithDefaultSelect(policy ?? undefined, config?.export?.nonReimbursableAccount),
+        () => getXeroBankAccounts(policy ?? undefined, config?.export?.nonReimbursableAccount),
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/52699, removing default value here cause inconsistent between "Export" page and "Xero bank account" page, and we handled it here https://github.com/Expensify/App/pull/52974